### PR TITLE
Adds scripts for some Temporary Items

### PR DESCRIPTION
**Pull request by neuromancerxi**
Adds Healing Salve I/II, Clear Salve I/II, Dusty Elixir

Lowered use timers from 4s to 1s (per bg wiki, 1s use time).
Clear Salve I can remove randomly up to two effects at random (https://translate.google.com/translate?hl=en&sl=ja&u=http://wiki.ffo.jp/html/20179.html&prev=search&pto=aue). Interpreted this to mean can remove between 1 and 2 effects.

**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/globals/items/tube_of_clear_salve_i.lua [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    end
+    
+    local toremove = math.random(1, 2)
+    local removed = 0
+    
+    repeat
+        if not removeStatus() then break end
+        toremove = toremove - 1
+        removed = removed + 1
+    until (toremove <= 0)
```
Comment (zach2good) [inline scripts/globals/items/tube_of_clear_salve_i.lua]: Ohhhh I didn't know Lua had repeat/until (do/while)!

File: scripts/globals/items/tube_of_clear_salve_ii.lua [hunk 2]
```diff
@@ -0,0 +1,44 @@
+-----------------------------------------
+-- ID: 5838
+-- Item: tube_of_clear_salve_ii
+-- Item Effect: Instantly removes all negative status effects from pet
+-----------------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/msg")
+
+function onItemCheck(target)
+    if (not target:hasPet()) then
```
Comment (zach2good) [inline scripts/globals/items/tube_of_clear_salve_ii.lua]: No need for these braces:
https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md#no-parentheses-unless-needed-to-clarify-order-of-operations

File: scripts/globals/items/dusty_elixir.lua [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+require("scripts/globals/msg")
+
+function onItemCheck(target)
+    local result = 0
+    local mHP = target:getMaxHP()
+    local cHP = target:getHP()
+    local mMP = target:getMaxMP()
+    local cMP = target:getMP()
+
+    if (mHP == cHP and mMP == cMP) then
```
Comment (zach2good) [inline scripts/globals/items/dusty_elixir.lua]: https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md#no-parentheses-unless-needed-to-clarify-order-of-operations

File: scripts/globals/items/tube_of_clear_salve_i.lua [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        if pet:delStatusEffect(tpz.effect.CURSE_II) then return true end
+        if pet:delStatusEffect(tpz.effect.CURSE) then return true end
+        if pet:delStatusEffect(tpz.effect.PARALYSIS) then return true end
+        if pet:delStatusEffect(tpz.effect.PLAGUE) then return true end
+        if pet:delStatusEffect(tpz.effect.POISON) then return true end
+        if pet:delStatusEffect(tpz.effect.DISEASE) then return true end
+        if pet:delStatusEffect(tpz.effect.BLINDNESS) then return true end
+        if pet:eraseStatusEffect() ~= 255 then return true end
+        return false
+    end
```
Comment (zach2good) [inline scripts/globals/items/tube_of_clear_salve_i.lua]: I know this section probably exists somewhere else, but there _has_ to be a better way of doing this...

File: scripts/globals/items/tube_of_healing_salve_i.lua [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    if (not target:hasPet()) then
+        return tpz.msg.basic.REQUIRES_A_PET
+    end
+    return 0
+end
+
+function onItemUse(target)
+    local percent = 50
+    local pet = target:getPet()
+    local totalHP = (pet:getMaxHP()/100)*percent
```
Comment (zach2good) [inline scripts/globals/items/tube_of_healing_salve_i.lua]: `local totalHP = (pet:getMaxHP() / 100) * percent`

**Review by zach2good — CHANGES_REQUESTED** [review]
Content seems fine, just some style dings. I'm also 1000% sure there is a better way to do that big counting status effect section.
```lua
-- A util of some sort
tpz.utils.removeNStatusEffects = function(target, nToRemove)
    local statusEffectTable = {...}
    local function removeStatus()
        for effect in statusEffectTable then
            if target:delStatusEffect(effect) then return true end
        end
        if target:eraseStatusEffect() ~= 255 then return true end
        return false
    end

    local removed = 0
    repeat
        if not removeStatus() then break end
        nToRemove= nToRemove- 1
        removed = removed + 1
    until (nToRemove<= 0)

    return removed
end

...


local pet = target:getPet()
local removed = tpz.utils.removeNStatusEffects(pet, math.random(1, 2))
return removed
```

Something like this? I dunno

File: scripts/globals/items/tube_of_clear_salve_i.lua [hunk 6]
```diff
@@ -1,44 +1,51 @@
...
+    if not target:hasPet() then
         return tpz.msg.basic.REQUIRES_A_PET
     end
     return 0
 end
 
 function onItemUse(target)
     local pet = target:getPet()
-    
+    local effects = {
```
Comment (TeoTwawki) [inline scripts/globals/items/tube_of_clear_salve_i.lua]:
```lua
    local effects =
    {
        tpz.effect.PETRIFICATION,
        tpz.effect.SILENCE,
        tpz.effect.BANE,
        tpz.effect.CURSE_II,
        tpz.effect.CURSE_I,
        tpz.effect.PARALYSIS,
        tpz.effect.PLAGUE,
        tpz.effect.POISON,
        tpz.effect.DISEASE,
        tpz.effect.BLINDNESS
    }
```
to conform to the [style guide](https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md#allman-braces)

**Review by zach2good — APPROVED** [review]

**Pull request merged**